encrypted-media/scripts/playback-temporary-waitingforkey.js

```diff
@@ -0,0 +1,90 @@
+function runTest(config) {
+
+    var testname = config.keysystem + ', successful playback, temporary, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', waitingforkey event';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
+                            audioCapabilities: [ { contentType: config.audioType } ],
+                            videoCapabilities: [ { contentType: config.videoType } ],
+                            sessionTypes: [ 'temporary' ] };
+
+    async_test( function( test ) {
+
+        var _video = config.video,
+            _mediaKeys,
+            _mediaKeySessions = [ ],
+            _mediaSource;
+
+        function onMessage(event) {
+            assert_any( assert_equals, event.target, _mediaKeySessions );
+            assert_true( event instanceof window.MediaKeyMessageEvent );
+            assert_equals( event.type, 'message');
+
+            assert_any( assert_equals,
+                        event.messageType,
+                        [ 'license-request', 'individualization-request' ] );
+
+            config.messagehandler( config.keysystem, event.messageType, event.message )
+            .then( function( response ) {
+
```

Comment:
There are lots of extra empty lines at the beginning and end of blocks. It seems inconsistent. I think we should remove them.